docs/nontech-explainer.md

```diff
@@ -0,0 +1,65 @@
+# Openness in Tech: An Explainer With NO Tech Jargon
+
+> What openness means, and why we care
+
+## Why We Need Openness
+
+In its broader sense, openness is mostly defined by negation. "Being open" means to "not be" closed, as boundaries allow to define closeness more clearly.
+
+The same idea goes for the more specific sense of openness we find in IT. IT mostly took shape during the cold war with the emergence of a "closed-world discourse[^1]" emphasizing security through obscurity and global surveillance. This discourse strongly influenced our vision of IT, and continues to do so today. We still tend to see code, innovation, and information as something we need to protect. We still think we need some kind of magical secret on which we could build benefits.
```

Comment:
> We still think we **need** some kind of magical secret on which we could build benefits.

Maybe restructure the phrase to start with benefits, and shift from need to can 

|--> We still think we can harness benefits by holding on some kind of magical secret.

Reply:
Great feedback, thx :+1: